CDS/js/test/TestDataStore.js

```diff
@@ -93,7 +95,33 @@ describe('Datastore',function(){
         //     assert.equal(datastore.get('media','mediaKey'),'somethingElse');
         // });
     });
+
+    describe('#substituteStrings', function() {
+        var substituteStub;
+
+        before(function() {
+            substituteStub = sinon.stub(datastore, 'substituteString', function(connection, value) {
```

Comment:
> So that if there is a problem in the substituteString function this test will still pass.

Doesn't this negate the purpose of the test though as we're testing the `datastore.substituteStrings` function?